Documentation/network/clustermesh/affinity.rst

```diff
@@ -32,15 +32,15 @@ remote endpoints if and only if all of local backends are not available or unhea
    metadata:
      name: rebel-base
      annotations:
-        io.cilium/global-service: "true"
+        io.cilium.service.global: "true"
```

Comment:
if we consider io.cilium.service as prefix and global is the name, should the annotation be io.cilium.service/global instead ? :thinking: 

https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set

Reply:
It should actually be `service.cilium.io/global: "true"`

See also `kubernetes.io/egress-bandwidth` example in https://docs.cilium.io/en/stable/gettingstarted/bandwidth-manager/ .
Will fix.

Reply:
oh well, looks all of them are messed up.. :/

Reply:
@borkmann if we are changing this then we should fix and make them all consistent as well as add a note in the upgrade docs mentioning that we are changing this label.